docs/Diagnostics/FreeBusyChecker/FreeBusyChecker.md

```diff
@@ -0,0 +1,145 @@
+# Hybrid-Free-Busy-Configuration-Checker
+
+To View this Project at GitHub! [GitHub Repository](https://github.com/MarcoLFrancisco/Hybrid-Free-Busy-Configuration-Checker)
+
+To Download the latest release: [FreeBusyChecker.ps1](https://github.com/MarcoLFrancisco/Hybrid-Free-Busy-Configuration-Checker/releases/download/Version1/FreeBusyChecker.ps1)
+
+To Provide Feedback about this tool: [Feedback Form](https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR2LVru-UswhJmHot_XEUrVVURFVMRkE5VUg4QUU0MEpNRjgxUExPVlBVOS4u)
+
+
+- This script does not make changes to current settings. It collects relevant configuration information regarding Hybrid Free Busy configurations on Exchange On Premises Servers and on Exchange Online, both for OAuth and DAuth.
+
+- This is a Beta Version. Please double check on any information provided by this script before proceeding to address any changes to your Environment. Be advised that there may be incorrect content in the provided output.
+
+Use: Collects OAuth and DAuth Hybrid Availability Configuration Settings Both for Exchange On Premises and Exchange Online
+
+Example Screen Output:
+
+![image](./image1.png)
+
+Example TXT Output:
+
+![image](./image2.png)
+
+Example HTML Output
+
+![image](./image3.png)
+
+Supported Exchange Server Versions:
+
+The script can be used to validate the Availability configuration of the following Exchange Server Versions: - Exchange Server 2013 - Exchange Server 2016 - Exchange Server 2019 - Exchange Online
+
+Required Permissions:
+
+- Organization Management
+- Domain Admins
+
+Please make sure that the account used is a member of the Local Administrator group. This should be fulfilled on Exchange servers by being a member of the Organization Management group. However, if the group membership was adjusted or in case the script is executed on a non-Exchange system like a management server, you need to add your account to the Local Administrator group.
+
+Other Pre Requisites:
+
+AD management Tools:
+
+If not available, they can be installed with the following command:
+
+  Install-WindowsFeature -name AD-Domain-Services -IncludeManagementTools
+
+Imports and Installs the following Modules (if not available):
+
+PSSnapin: microsoft.exchange.management.powershell.snapin
+Module  : ActiveDirectory Module
+Module  : ExchangeOnlineManagement Module
+
+
+Syntax:
+
+    FreeBusyChecker.ps1
```

Comment:
use the same format that is in `SourceSideValidations.md`

Reply:
I believe it is as intended now.